mindsdb/integrations/handlers/neuralforecast_handler/neuralforecast_handler.py

```diff
@@ -0,0 +1,99 @@
+import os
+import string
+import random
+from mindsdb.integrations.libs.base import BaseMLEngine
+from mindsdb.integrations.handlers.statsforecast_handler.statsforecast_handler import (
+    infer_frequency,
+    transform_to_nixtla_df,
+    get_results_from_nixtla_df,
+)
+from neuralforecast import NeuralForecast
+from neuralforecast.models import NHITS
+from neuralforecast.auto import AutoNHITS
+from ray.tune.search.hyperopt import HyperOptSearch
```

Comment:
Missing `ray` in the `requirements.txt` file

Reply:
Good spot.

Reply:
My bad, turns out NeuralForecast includes `ray[tune]` by default, we should revert this.